app/models/ability.rb

```diff
@@ -11,6 +11,21 @@ def initialize(current_user, current_group=nil)
         person.groups.include?(current_group)
       end
 
+      can :manage, Person do |person|
+        permitted_flag = false
+        permitted_flag = true if person == current_user
+        permitted_flag = true if Membership.organizer.exists?(group: current_group, person: current_user)
```

Comment:
for my own edification: are we allowed to write `current_user`? i thought we had to write `current_person`?